.gitignore

```diff
@@ -2,7 +2,21 @@
 # Files already tracked by Git are not affected.
 # See: https://git-scm.com/docs/gitignore
 
+# Binaries
+*.exe
+*.exe~
+*.dll
+*.so
+*.dylib
+
 ## Python ##
 __pycache__
 .mypy_cache
 .python-version
+.venv
+
+# Test binary, built with `go test -c`
+*.test
+
+# Environment variables
+.env
```

Comment:
Please keep sections sorted alphabetically, so move this section up above and match our preferred formatting: `## Environment variables ##`

Reply:
@dav3r resolved with [3add000](https://github.com/cisagov/con-pca-tasks/pull/1/commits/3add000073e20deba5ce69e868236d0bde794655)

Reply:
Oops, missed it. @dav3r resolved with [8c33d7c](https://github.com/cisagov/con-pca-tasks/pull/1/commits/8c33d7c4e91f24e6457a0fa1941c94083a5e8111)